src/meta/src/hummock/manager/compaction_group_manager.rs

```diff
@@ -103,6 +103,12 @@ impl<S: MetaStore> HummockManager<S> {
             == Some(true);
         let mut pairs = vec![];
         if let Some(mv_table) = mv_table {
+            for internal_table_id in &internal_tables {
+                self.metrics
+                    .mv_info
+                    .with_label_values(&[&mv_table.to_string(), &internal_table_id.to_string()])
+                    .set(1)
+            }
```

Comment:
mv_info is maintained incrementally here, which is not a good idea. See the discussion [here](https://github.com/risingwavelabs/risingwave/pull/8896#pullrequestreview-1366253017).

Reply:
After some offline discussion will @zwang28 , we already has the [mapping relations](https://github.com/risingwavelabs/risingwave/blob/f51c14dcd3c00d32013379874c12ff111b3c8afb/src/meta/src/model/stream.rs#L43) from mv to it's internal tables, and we can get the map by the [all_table_ids](https://github.com/risingwavelabs/risingwave/blob/f51c14dcd3c00d32013379874c12ff111b3c8afb/src/meta/src/model/stream.rs#L479) interface.

So I think maybe we don't need to maintain the mapping in compaction_group_manager, I will try to calculate the mv size every time `trigger_version_stat` is called.

Reply:
How about metrics other than the size of a MV, like memory consumption? Are we also going to emit both per actor and per mv usage?

Reply:
Sure~ metrics like memory consumption and throughput will be done in next PR, I'm thinking about how to monitor it. This PR is only for the size.